back-end/helpers/transcriptDataFormatter.js

```diff
@@ -1,70 +1,70 @@
-
 export default class transcriptDataFormatter {
-  /**
-   * Cleans up and formats raw transcipt data
-   * @param {Array} transcript
-   * @returns
-   */
-  static cleanData(transcript) {
-    var res = [];
-    var block;
+    /**
+     * Cleans up and formats raw transcipt data
```

Comment:
small typo: "transcipt" should be "transcript"

Reply:
also, on that note, avoid using the 'var' keyword and use 'let' and 'const' instead.

Reply:
i think marco wrote that function. i'll let him know so  he can make the changes.

Reply:
I just fixed it now good catch Chelsea!